lib/ci/lib/rails_upgrade_analyzer/github_proxy.rb

```diff
@@ -0,0 +1,43 @@
+module RailsUpgradeAnalyzer
+  class AuthenticationError < RuntimeError
+  end
+
+  class GithubProxy
+    def self.access_token
+      ENV.fetch("GITHUB_TOKEN") do
```

Comment:
I mentioned this elsewhere, but probably best to pass in a reference to `ENV` rather than hardcoding a dependency on it here...